chapter-2/README-pt.md

```diff
@@ -0,0 +1,122 @@
+# Capítulo 2 :: Desafios da Aplicação Cloud Native
+
+---
+_🌍 Disponível em_: [English](README.md) | [中文 (Chinese)](README-zh.md) | [Português (Portuguese)](README-pt.md)
+
+> **Nota:** Trago a você pela fantástica comunidade cloud-native e seus [ 🌟 contribuidores](https://github.com/salaboy/platforms-on-k8s/graphs/contributors)!
+
+---
+
+Neste tutorial, vamos instalar a `Conference Application` em um cluster Kubernetes KinD local, usando o Helm.
+
+> [!NOTE]
+> Os Helm Charts podem ser publicados em repositórios de Helm Chart ou, a partir da versão 3.7, como contêineres OCI em container registries.
+
+## Criando um cluster local com o Kubernetes KinD
+
+> [!Important]
+> Certifique-se de ter os pré-requisitos para todos os tutoriais. Você pode encontrá-los [aqui](../chapter-1/README.md#pre-requisites-for-the-tutorials).
+
+Utilize o comando abaixo para criar um Cluster KinD com três worker nodes e 1 Control Plane.
+
+```shell
+cat <<EOF | kind create cluster --name dev --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+- role: worker
+- role: worker
+- role: worker
+EOF
+```
+
+Uma vez que temos nosso cluster e nosso Ingress Controller instalados e configurados, podemos prosseguir com a instalação de nossa aplicação.
```

Comment:
Hello @salaboy, where is the step to install the Ingress Controller, shouldn't it be here?